recipes/gottcha/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: gottcha
+  version: "1.0"
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  fn: 23a7f0df7b5203a56354ed4926d40c19e7e8f6bd.tar.gz
+  url: https://github.com/LANL-Bioinformatics/GOTTCHA/archive/23a7f0df7b5203a56354ed4926d40c19e7e8f6bd.tar.gz
+  md5: ce904c44f9a0f7a45a81268967e143cd
+
+requirements:
+  build:
+    - unzip
+    - bwa
+    - dmd
+    - perl-threaded
+    - perl-app-cpanminus
+    - perl-module-build 
+    - perl-tie-ixhash
+    - perl-yaml
+  run:
+    - perl-threaded
+    - bwa
+
+test:
+  commands:    
+    - perl gottcha.pl --help | grep -q '^VERSION: 1.0b'; echo $?
```

Comment:
Can you remove `perl` from the test. gottcha.pl should be at the PATH, correct. So you don't need it.